eth/tracers/api_blocktrace.go

```diff
@@ -211,6 +213,13 @@ func (api *API) getBlockTrace(block *types.Block, env *traceEnv) (*types.BlockTr
 		}
 	}
 
+	// build dummy per-tx deletion proof
```

Comment:
Why do we need this? We don't actually provide per-tx deletion proofs?

Reply:
cc @noel2004

Reply:
we can keep it as it is, and refactor in the future? wdyt @lispc

Reply:
ok